src/app/Home/components/HomeComponent.js

```diff
@@ -1,14 +1,71 @@
 /* eslint-disable react/destructuring-assignment */
 import React, { Component } from 'react';
-import TestSaga from './testSaga';
+import Modal from './onBoardingModal';
+import LoginModal from './logInPointsModal';
 
 // eslint-disable-next-line react/prefer-stateless-function
 export class HomeComponent extends Component {
-  render() {
-    return (
-      <TestSaga />
-    );
+  constructor() {
+    super();
+
+    this.state = {
+      isShowing: false,
+      isShowingLoginPoints: false,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      isShowing: true,
+    });
```

Comment:
I don't think we need this it seems you want to start with isShowing true you can set that in the constructor